avail/src/main/kotlin/avail/persistence/cache/record/StylingRecord.kt

```diff
@@ -227,13 +240,24 @@ class StylingRecord
 				repeat(usagesOrZero) {
 					val useStart = previousUseEnd + binaryStream.unvlqInt()
 					val use = useStart until useStart + length
-					usesToDeclarations.add(use to decl)
+					usesToDeclarations.add(use to declaration)
 					previousUseEnd = use.last + 1
 				}
 			}
-			previousDeclarationEnd = declStart + length
+			previousDeclarationEnd = declarationStart + length
 		}
 		variableUses = usesToDeclarations
+
+		val numUnusedDeclarations = binaryStream.unvlqInt()
```

Comment:
Following this `vql` stuff today, I'm learning about https://en.wikipedia.org/wiki/Variable-length_quantity 🙏